src/react-components/ui-root.js

```diff
@@ -1306,7 +1306,7 @@ class UIRoot extends Component {
             label: "Create Room",
             icon: HomeIcon,
             onClick: () =>
-              this.showNonHistoriedDialog(LeaveRoomDialog, {
+              this.showNonHistoriedDialog(LeaveRoomModal, {
```

Comment:
Might also be good to flow thru this modal as a confirmation step for visiting rooms via a room link. I've accidentally clicked one before and it would have been nice to back out of it.
![image](https://user-images.githubusercontent.com/4072106/98879493-88fe4380-243a-11eb-8217-80c45a8f3a5b.png)

Reply:
Hmm, that would break the flow of things like Chamberlain's lighthouse experience though. I don't know that I agree with this, but I'd be happy to follow up discussion in another issue.